openassessment/templates/openassessmentblock/message/oa_message_incomplete.html

```diff
@@ -1,56 +1,54 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">
-        {% trans "Assignment in progress" %}
-    </h3>
+    <h4 class="message__title">{% trans "Progress" %}</h4>
```

Comment:
@catong I'm OK with Status or Progress since the Heading should describe the content that follows it.  If we change it to In Progress, then it crosses the line into not describing the content that follows but actually becoming content.  I also wanted to choose something generic enough to be used in each step.  If we call it "In Progress" then it may be "Complete" or "Not yet started" for instance.  For someone who relies on anchoring, or browsing by heading, they won't have something consistent to look for.